README.md

```diff
@@ -84,6 +84,44 @@ This can be run on a regular basis using a scheduler like cron. We recommend an
 - If using cron as a scheduler, [lockrun](http://unixwiz.net/tools/lockrun.html) can be used to prevent multiple instance of the same job running simultaneously.
 - If using a queue-based scheduler like Celery Beat, the `SyncManager` class contains `is_queued` and `is_running` extension points which could be used to implement a lock strategy.
 
+## Signals
+
+`translation_imported`
+
+This signal is sent when a translation from RWS LanguageCloud is successfully imported.
+
+**Arguments:**
+
+- **sender:** `LanguageCloudProject`
+- **instance:** The specific `LanguageCloudProject` instance.
+- **translation_source_object:** The page or snippet instance that this translation is for.
+
+Here’s how you could use it to send Slack notifications.
+
+```python
+from wagtail_localize_rws_languagecloud.signals import translation_imported
+import requests
+
+# Let everyone know when translations come back from RWS LanguageCloud
+def send_to_slack(sender, instance, translation_source_object):
+    url = (
+        "https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX"
+    )
+
+    values = {
+        "text": f'New translations for "{translation_source_object}" have been imported.',
```

Comment:
Obviously its up to the user how to handle the signal, but it would be nice if our example code showed how to put the URL into the message as that is probably necessary for a minimal useful slack message. Useful reference point: https://github.com/zerolab/wagtail-localize-rws-languagecloud/blob/ffe99fd3e6255e649b7407711d9a01b1e97c96ed/wagtail_localize_rws_languagecloud/emails.py#L23-L32

We could also maybe provide some guidance on where this signal handling code should live in your application, or link to some docs? https://docs.djangoproject.com/en/3.2/topics/signals/#connecting-receiver-functions

Reply:
Good points. I will update the docs.

Reply:
Updated the docs.

I opted to just include the edit URL in the text. I didn't want to go for something too fancy/complex for the example.